poc/vdaf_prio3.py

```diff
@@ -115,16 +127,19 @@ def prep_next(Prio3, prep, prep_msg):
     @classmethod
     def prep_shares_to_prep(Prio3, _agg_param, prep_shares):
         # Unshard the verifier shares into the verifier message.
-        verifier = Prio3.Flp.Field.zeros(Prio3.Flp.VERIFIER_LEN)
+        verifiers = Prio3.Flp.Field.zeros(
+            Prio3.Flp.VERIFIER_LEN * Prio3.PROOFS)
         k_joint_rand_parts = []
         for (verifier_share, k_joint_rand_part) in prep_shares:
```

Comment:
```suggestion
        for (verifiers_share, k_joint_rand_part) in prep_shares:
```

Reply:
I think this is reflected in the latest version.